src/views/CharacterListView.js

```diff
@@ -3,19 +3,23 @@ import { connect } from "react-redux";
 
 import { CharacterList } from "../components";
 // import actions
+import { getStarwarsCharacters } from "../actions";
 
 class CharacterListView extends React.Component {
   constructor() {
     super();
   }
 
   componentDidMount() {
-    // call our action
+    console.log(this.props)
+    this.props.getStarwarsCharacters()
+    
   }
 
-  render() {
+  render() {console.log(this.props)
     if (this.props.fetching) {
       // return something here to indicate that you are fetching data
+      return <h3>Fetching Your Starwars Data...</h3>
```

Comment:
I like that you specifically indicated "Starwars Data" because it lets users know what they are waiting for. It's a good idea to keep this up.